pages/about.html

```diff
@@ -5,8 +5,39 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
+  <link href="../styles/style.css" rel="stylesheet"> 
 </head>
+
 <body>
-  
+  <header>
+    <h1>
+      About Me.
+    </h1>
+    <p>
+      Human. Hmong. Activist. Engineer.
+  </p>
+  </header>
+  <nav>
+    <ul>
+      <li><a href="index.html">Home</a></li>
+      <li><a href="portfolio.html">My Work</a></li>
+    </ul>
+  </nav>
+
+  <article class="my-story">
+    <h3>Nyob zoo and hello!</h3>
```

Comment:
This should have been a `<h2>` tag (you shouldn't skip heading levels).  It's pretty easy to re-style the size of tags with CSS.

```css
h2 {
    font-size: 10px; /* Or whatever. */
}
```